lua_test.go

```diff
@@ -0,0 +1,20 @@
+package lua
+
+import (
+	"regexp"
+	"testing"
+)
+
+// https://github.com/Shopify/go-lua/pull/63
```

Comment:
This is probably unnecessary, going by the pull requests I submitted. (i.e., git blame will lead back to the PR and/or relevant issues).